### PR TITLE
feat: Define license assign endpoint on Gateway

### PR DESCRIPTION
**Pull request by sameenfatima78**
This PR adds a definition for license-manager assign endpoint on the Omnibus API to expose it to Gateway. 

Host variable is also added to pick up host value for the service. https://github.com/edx/edx-internal/pull/9905

**JIRA**: https://2u-internal.atlassian.net/browse/ENT-7995

### CODE REVIEW

File: scripts/aws/deploy.py [hunk 1]
```diff
@@ -47,6 +47,8 @@
                         help="Location of enterprise catalog IDA for request routing")
     parser.add_argument('--authoring-host', required=True,
                         help="Location of Studio for authoring request routing")
+    parser.add_argument('--license-manager-host', required=True,
```
Comment (zamanafzal) [inline scripts/aws/deploy.py:50]: For few args like `--enterprise-host` it is marked as `required=False`, how we are figuring this out?

**Review by zamanafzal — APPROVED** [review]
LGTM 👍🏻

**Review by hamzawaleed01 — APPROVED** [review]

**Comment by bszabo** [comment]
@sameenfatima78 Good day! Could you please have a look at the build status for the `stage-edx-api` GoCD pipeline stage? This PR appears to be generating the following error:

```
botocore.errorfactory.BadRequestException: An error occurred (BadRequestException) when calling the PutRestApi operation: Errors found during import:
	Unable to create model for 'body_1': Model name must be alphanumeric: body_1
```

I'm interested in merging unrelated changes, so please let me know if I can be of assistance, or, if that's not needed, when I should try my merge.

**Comment by sameenfatima78** [comment]
@bszabo Hi, Bernard. I'm currently looking into this but haven't made much progress yet. Will let you know if I'm able to figure out the cause otherwise I can revert my changes to unblock you, if required.

**Comment by sameenfatima78** [comment]
@bszabo Just pushed a fix and the pipeline is all green now.

**Pull request merged**